python/hackathon_app/server/main.py

```diff
@@ -101,9 +101,10 @@ def user_info():
 def verify_google_token():
     try:
         # Specify the CLIENT_ID of the app that accesses the backend:
+        # breakpoint()
         body = flask.request.json
         idinfo = id_token.verify_oauth2_token(
-            body["Zi"]["id_token"],
+            body["tokenId"],
```

Comment:
not sure how/why/when this broke but I only found it by manually exercising the google sign in via UI. we don't have any test coverage at this controller level :-( not quite sure how we'd test it though - more a dependency on the frontend library (`react-google-login`) sending in a certain payload